plugin-cgroupv2/src/k8s/probe.rs

```diff
@@ -93,6 +93,19 @@ impl K8SProbe {
 }
 
 impl alumet::pipeline::Source for K8SProbe {
+    /// # Function
+    ///
+    /// The `poll` function allows to obtain a measurement determined by a data point, format it and transmit it
+    ///
+    /// # Parameters
+    ///
+    /// - `measurements` : An accumulator stores measured data points
+    /// - `timestamp` : Time recorded at the time of measurement
+    ///
+    /// # Return
+    ///
+    /// Success or and error which can occur during `Source::poll`
+    ///
```

Comment:
Like structures, functions should be documented in a "direct" approach. We know it's a function, don't say it in the title.
Furthermore, `poll` is defined in `alumet::pipeline::Source`, there is no need to document the same thing in its implementation. If you have specific things to say about _this implementation of `poll`_, write them in the doc. Otherwise, I think that it's unnecessary to document it.